container.nix

```diff
@@ -17,6 +17,14 @@ let
       property = "AddCapability";
     };
 
+    addHosts = quadletUtils.mkOption {
+      type = types.listOf types.str;
+      default = [ ];
+      example = ["hostname:192.168.10.11"];
+      description = "--add-host";
+      property = "AddHost";
```

Comment:
Support for `AddHost` was added in https://github.com/containers/podman/pull/23713

It's available since Podman v5.3.1. The current version in the 24.11 channel is 5.2.3. This makes me wonder if it's worth adding a warning if the version is less than 5.3.1. 
But this creates a maintaining burden, to keep track of all the podman versions... 🤔

Reply:
LGTM given the default value is safe.

There are too many ways the config could be bad and surely we can't catch all. Thus I don't see this project should be responsible for config check beyond what's trivial to do.